core/src/main/java/hudson/Functions.java

```diff
@@ -513,7 +513,15 @@ public static Map getEnvVars() {
     public static boolean isWindows() {
         return File.pathSeparatorChar==';';
     }
-    
+
+    public static boolean isMac() {
+        String osType = System.getProperty("os.name");
+        if (osType == null || osType.isEmpty() || !osType.toLowerCase().contains("mac os x")) {
```

Comment:
I assume we do not need to worry about the [Turkish and other exotic locale](https://stackoverflow.com/questions/11063102/using-locales-with-javas-tolowercase-and-touppercase) specific issues? 
```suggestion
        if (osType == null || osType.isEmpty() || !osType.toLowerCase(Locales.ENGLISH).contains("mac os x")) {
```